static/index.html

```diff
@@ -12,6 +12,7 @@
     <meta property="og:description" content="All your favorite LBRY content in your browser." />
     <meta property="og:image" content="/og.png" />
     <!-- @endif -->
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
@kauffj You said not to build a mobile layout, but you didn't say _not_ to create a new desktop layout that just happens to look good on mobile too.